packages/seacas/libraries/ioss/src/exodus/Ioex_DatabaseIO.C

```diff
@@ -1550,12 +1550,12 @@ namespace Ioex {
       // have already created the output database and populated the set/block names. At this point,
       // it is too late to change the size of the names stored on the output database... (I think...
       // try changing DIM_STR_NAME value and see if works...)
-      if (name_length >= (size_t)maximumNameLength) {
+      if (name_length > (size_t)maximumNameLength) {
```

Comment:
Prefer C++ casts: either `static_cast<size_t>(maximumNameLength)` or `size_t(maximumNameLength)`.

Reply:
Agreed.

Reply:
You can check and see but I am pretty sure that `size_t(maximumNameLength)` is just a C-style cast identical to `(size_t)maximumNameLength`.  For a C++11 safe conversion use `size_t{maximumNameLength}`